### DISCUSSION TITLE
Welcome to excel_comp_team_106 Discussions!

### DISCUSSION THREAD

**Discussion by codabytez** [zuri-training/excel_comp_team_106#13] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Ola2nji** [comment]
I'm name is Odejobi Yusuf, I am a Product designer based in Lagos , Nigeria.

**Reply by Mardiyyahae** [reply]
I am Adeniyi-Edun Meridiyat a product designer
